99_master-dynamic-system-reserved-calc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-remove-systemd-reserved-mem-cpu
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,IyEvYmluL2Jhc2gKCnNldCAtZQoKTk9ERV9TSVpFU19FTlY9JHtOT0RFX1NJWkVTX0VOVjotL2V0Yy9ub2RlLXNpemluZy5lbnZ9CgpmdW5jdGlvbiBkeW5hbWljX25vZGVfc2l6aW5nIHsKICAgIHJtIC1mICR7Tk9ERV9TSVpFU19FTlZ9CiAgICBlY2hvICJTWVNURU1fUkVTRVJWRURfTUVNT1JZPTFHaSIgPj4gJHtOT0RFX1NJWkVTX0VOVn0KICAgIGVjaG8gIlNZU1RFTV9SRVNFUlZFRF9DUFU9NTAwbSIgPj4gJHtOT0RFX1NJWkVTX0VOVn0KfQoKZnVuY3Rpb24gc3RhdGljX25vZGVfc2l6aW5nIHsKICAgIHJtIC1mICR7Tk9ERV9TSVpFU19FTlZ9CiAgICBlY2hvICJTWVNURU1fUkVTRVJWRURfTUVNT1JZPTBHaSIgPj4gJHtOT0RFX1NJWkVTX0VOVn0KICAgIGVjaG8gIlNZU1RFTV9SRVNFUlZFRF9DUFU9MG0iID4+ICR7Tk9ERV9TSVpFU19FTlZ9Cn0KCmlmIFsgJDEgPT0gInRydWUiIF07IHRoZW4KICAgIGR5bmFtaWNfbm9kZV9zaXppbmcKZWxpZiBbICQxID09ICJmYWxzZSIgXTsgdGhlbgogICAgc3RhdGljX25vZGVfc2l6aW5nCmVsc2UKICAgIGVjaG8gIlVucmVjb25nbml6ZWQgY29tbWFuZCBsaW5lIG9wdGlvbi4gVmFsaWQgb3B0aW9ucyBhcmUgXCJ0cnVlXCIgb3IgXCJmYWxzZVwiIgpmaQoK
```

Comment:
I don't know if there's an 'official' way of doing this, but can we store the real source file in git, and generate this file at build time?
In git: 
```
$ cat dynamic-system-reserved-calc.sh
#!/bin/bash

set -e

NODE_SIZES_ENV=${NODE_SIZES_ENV:-/etc/node-sizing.env}

function dynamic_node_sizing {
    rm -f ${NODE_SIZES_ENV}
    echo "SYSTEM_RESERVED_MEMORY=1Gi" >> ${NODE_SIZES_ENV}
    echo "SYSTEM_RESERVED_CPU=500m" >> ${NODE_SIZES_ENV}
}

function static_node_sizing {
    rm -f ${NODE_SIZES_ENV}
    echo "SYSTEM_RESERVED_MEMORY=0Gi" >> ${NODE_SIZES_ENV}
    echo "SYSTEM_RESERVED_CPU=0m" >> ${NODE_SIZES_ENV}
}

if [ $1 == "true" ]; then
    dynamic_node_sizing
elif [ $1 == "false" ]; then
    static_node_sizing
else
    echo "Unrecongnized command line option. Valid options are \"true\" or \"false\""
fi

$ cat 99_master-dynamic-system-reserved-calc.yaml.in
...
    storage:
      files:
      - contents:
          source: data:text/plain;charset=utf-8;base64,@BASE64_CONTENT@
...
```
and replace `@BASE64_CONTENT@` with `cat dynamic-system-reserved-calc.sh | base64 -w0`

Reply:
I just noticed a `Unrecongnized` typo (`Unrecognized`)

Reply:
+1 for this. Easier to search/understand.